PowerEditor/src/WinControls/DockingWnd/DockingCont.cpp

```diff
@@ -1094,8 +1076,8 @@ void DockingCont::drawTabItem(DRAWITEMSTRUCT *pDrawItemStruct)
 		if ((hImageList != NULL) && (iPosImage >= 0))
 		{
 			// Get height of image so we
-			IMAGEINFO	info		= {};
-			RECT &		imageRect	= info.rcImage;
+			IMAGEINFO info = {};

```

Comment:
for consistency I would use `IMAGEINFO info{};`, same for `TCITEM` variables below